forge-gui/res/cardsfolder/upcoming/gix_yawgmoth_praetor.txt

```diff
@@ -0,0 +1,12 @@
+Name:Gix, Yawgmoth Praetor
+ManaCost:1 B B
+Types:Legendary Creature Phyrexian Praetor
+PT:3/3
+T:Mode$ DamageDone | ValidSource$ Creature | ValidTarget$ Opponent | CombatDamage$ True | OptionalDecider$ TriggeredSourceController | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ Whenever a creature deals combat damage to one of your opponents, its controller may pay 1 life. If they do, they draw a card.
+SVar:TrigDraw:AB$ Draw | Cost$ PayLife<1> | NumCards$ 1 | Defined$ TriggeredSourceController
```

Comment:
This will always have "You" pay the life
Might need to use UnlessCost

Reply:
UnlessCost seems right... another option might be TriggerController in line 5

Reply:
No, changing TriggerController would be against rules

Reply:
^ Thank you both for the feedback. I just tested the UnlessCost/UnlessSwitched solution as suggested above and it is now correctly prompting the creature's controller to pay life to draw a card. The trigger controller remains the controller of Gix himself, so no rules spaghetti. Thanks again